staging/src/k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset/clientset.go

```diff
@@ -71,17 +71,16 @@ func NewForConfig(c *rest.Config) (*Clientset, error) {
 		}
 		configShallowCopy.RateLimiter = flowcontrol.NewTokenBucketRateLimiter(configShallowCopy.QPS, configShallowCopy.Burst)
 	}
-	var cs Clientset
-	var err error
-	cs.apiextensionsV1beta1, err = apiextensionsv1beta1.NewForConfig(&configShallowCopy)
-	if err != nil {
-		return nil, err
-	}
-	cs.apiextensionsV1, err = apiextensionsv1.NewForConfig(&configShallowCopy)
+
+	factory, err := rest.RESTClientFactoryFor(&configShallowCopy)
 	if err != nil {
 		return nil, err
 	}
 
+	var cs Clientset
+	cs.apiextensionsV1beta1 = apiextensionsv1beta1.New(factory.NewForGV("Apiextensions.k8s.io", "V1beta1"))
```

Comment:
capital case looks odd

Reply:
fixed in https://github.com/kubernetes/kubernetes/pull/105432/commits/ef6516fa9d78b724a27856c75a120e84520dcbe0